libmachine/cert/cert.go

```diff
@@ -195,12 +197,16 @@ func ValidateCertificate(addr, caCertPath, serverCertPath, serverKeyPath string)
 		return false, ErrValidatingCert{err}
 	}
 
+	log.Debugf("Configuration: +%v", tlsConfig)
+
 	dialer := &net.Dialer{
-		Timeout: time.Second * 2,
+		Timeout: time.Second * 5,
+		DualStack: true,
```

Comment:
@nathanleclaire none ;-) I was just trying random things with the dialer but it didn't help